Detector/DetStudies/tests/options/upstreamMaterial_inclinedEcal.py

```diff
@@ -0,0 +1,79 @@
+from Gaudi.Configuration import *
+
+# Data service
+from Configurables import FCCDataSvc
+podioevent = FCCDataSvc("EventDataSvc")
+
+# DD4hep geometry service
+from Configurables import GeoSvc
+geoservice = GeoSvc("GeoSvc", detectors=[ 'file:Detector/DetFCChhBaseline1/compact/FCChh_DectEmptyMaster.xml',
+                                          'file:Detector/DetFCChhECalInclined/compact/FCChh_ECalBarrel_upstream.xml'
+],
+                    OutputLevel = INFO)
+
+# Geant4 service
+# Configures the Geant simulation: geometry, physics list and user actions
+from Configurables import SimG4Svc
+geantservice = SimG4Svc("SimG4Svc", detector='SimG4DD4hepDetector', physicslist="SimG4FtfpBert", actions="SimG4FullSimActions")
+geantservice.g4PostInitCommands += ["/run/setCut 0.1 mm"]
+
+# Geant4 algorithm
+# Translates EDM to G4Event, passes the event to G4, writes out outputs via tools
+# and a tool that saves the calorimeter hits
+from Configurables import SimG4Alg, SimG4SaveCalHits, SimG4SingleParticleGeneratorTool
+saveecaltool = SimG4SaveCalHits("saveECalBarrelHits",readoutNames = ["ECalBarrelEta"])
+saveecaltool.positionedCaloHits.Path = "ECalBarrelPositionedHits"
+saveecaltool.caloHits.Path = "ECalBarrelHits"
+from Configurables import SimG4SingleParticleGeneratorTool
+pgun=SimG4SingleParticleGeneratorTool("SimG4SingleParticleGeneratorTool",saveEdm=True,
+                                      particleName = "e-", energyMin = 10000, energyMax = 10000, etaMin = 0, etaMax = 0,
+                                      OutputLevel = DEBUG)
+# next, create the G4 algorithm, giving the list of names of tools ("XX/YY")
+geantsim = SimG4Alg("SimG4Alg",
+                    outputs= ["SimG4SaveCalHits/saveECalBarrelHits"],
+                    eventProvider = pgun,
+                    OutputLevel = DEBUG)
+
+
+from Configurables import CreateCaloCells
+createcellsBarrel = CreateCaloCells("CreateCaloCellsBarrel",
+                                    doCellCalibration=False,
+                                    addCellNoise=False, filterCellNoise=False)
+createcellsBarrel.hits.Path="ECalBarrelHits"
+createcellsBarrel.cells.Path="ECalBarrelCells"
+
```

Comment:
Maybe we can use one of the simulation files already produced in other tests, not to run the simulation again. We would just need to save the particle in the simulation job I guess. Or do you run new simulations as you need higher statistics?

Reply:
Hi Jana, but this sim must run with the cryostat set as sensitive which in principle we could activate in other studies, but then we would need to always check if cryo==0 to filter the energy deposited in LAr.

Reply:
OK, I see. I forgot about that. Sure, than it makes sense to have a new simulations for that.